src/test/kotlin/de/bund/digitalservice/useid/config/TestApplicationProperties.kt

```diff
@@ -0,0 +1,17 @@
+package de.bund.digitalservice.useid.config
+
+import de.bund.digitalservice.useid.util.YamlPropertySourceFactory
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.boot.context.properties.EnableConfigurationProperties
+import org.springframework.context.annotation.PropertySource
+
+@EnableConfigurationProperties(TestApplicationProperties::class)
+@PropertySource("classpath:application.yaml", factory = YamlPropertySourceFactory::class)
+@ConfigurationProperties(prefix = "application")
+class TestApplicationProperties {
+    var staging: StagingProperties? = null
```

Comment:
why do we set the value as null? is it possible with `lateinit` as well?

Reply:
Somehow that leads to errors on initialisation. 
 
<img width="892" alt="Screenshot 2022-08-15 at 10 13 41" src="https://user-images.githubusercontent.com/5744476/184600744-efb2a017-9f5f-4ccf-8222-8cbec65d35bb.png">

Reply:
About your package idea: 
The YAML factory could be part of a "one-time" package which is not touched anymore. The TestApplicationProperties can be reused for any config that should be available in any tests so I would like to keep that in the config package.

Reply:
that would be great if we can separate that and also for the TestApplicationProperties, maybe we could add an information about the purpose of the file?

Reply:
Like that?